sql2.txt

```diff
@@ -17,29 +29,58 @@ may have ordered the same product in several different orders.  So the total qua
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
 
+SELECT Customers.CustomerName, Products.ProductName, SUM(OrderDetails.Quantity) AS TotalQuantity
+FROM Customers
+JOIN Orders on Customers.CustomerID = Orders.CustomerID                                       
+JOIN OrderDetails on Orders.OrderID = OrderDetails.OrderID
+JOIN Products on OrderDetails.ProductID = Products. ProductID
+GROUP BY Customers.CustomerID, OrderDetails. ProductID; 
+
+
 
 
 4. Extend the previous query to give the total price as well as the customer name, product name, 
 and total quantity.  Here you will have
 (SUM(Quantity) * Price) AS TotalPrice.  Paste your SQL statement below.
 
+SELECT Customers.CustomerName, Products.ProductName, SUM(OrderDetails.Quantity) AS TotalQuantity, (SUM(OrderDetails.Quantity)*Products.Price) AS TotalPrice
+FROM Customers
+JOIN Orders on Customers.CustomerID = Orders.CustomerID                                       
+JOIN OrderDetails on Orders.OrderID = OrderDetails.OrderID
+JOIN Products on OrderDetails.ProductID = Products. ProductID
+GROUP BY Customers.CustomerID, OrderDetails. ProductID; 
+
 
 5. Add a new Order to the Orders table.  You will have to specify the ID of an existing customer
 to satisfy the foreign key constraint.  You will also have to specify the ID of an existing
 employee.  Paste your SQL statement below.
 
+INSERT INTO Orders                                                                               
+(OrderID, CustomerID, EmployeeID, OrderDate, ShipperID)                                                      
+Values (10448, 76, 4, "1996-7-10", 2);  
 
 6. Add three OrderDetails.  These should specify the ID of the new Order you just created.  For each,
 you will have to specify the ID of an existing product.  Paste your SQL statement below.
 
+INSERT INTO OrderDetails
+("OrderDetailID", "OrderID", "ProductID", "Quantity")    
```

Comment:
Super close! You got all of the hard parts. 

You just don't need quotes around any of the column names.